app.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "thing",
+  "scripts": {},
+  "env": {
+    "CERTIFICATE": {
```

Comment:
We shouldn't move the certs or IDP, URN, and SP stuff to pr apps. It won't work as they won't be registered in touchstone.

Reply:
👍 Removing them.